### DISCUSSION TITLE
Enhancement: Allow user to hide default system fields in the collections list view

### DISCUSSION THREAD

**Discussion by mahirchavda** [pocketbase/pocketbase#941] [Q&A] [answered]
Currently, I'm not able to hide id, created, updated field in the pockabase collections UI. Also it would be great if i can rearrange  default system fields as well.

I'm using PocketBase v0.8.0-rc2

**Comment by ganigeorgiev** [comment, marked as answer]
For now this is not possible, because system fields are not part of the `Collection.Schema` and they cannot be rearranged. The same apply for the "Toggle visibility" option, although here it is simpler since it is only in the UI and I'll consider adding the system fields in the dropdown in future versions.

**Reply by mahirchavda** [reply]
Thanks 👍👍

**Reply by ganigeorgiev** [reply]
@mahirchavda Option to toggle the visibility of system fields (with the exception of the `id`) was added in the `rc` branch. It will be available with the final v0.8.0 release.

**Reply by mahirchavda** [reply]
Nice. Thank You.

**Reply by laszlo1337** [reply]
> @mahirchavda Option to toggle the visibility of system fields (with the exception of the `id`) was added in the `rc` branch. It will be available with the final v0.8.0 release.

What was the reason to disallow hiding ID? it's very polluting and unnecessary, I cannot ever semantically differentiate records by looking at the ID, rather I am looking at name or slug or whatever. Kinda sucks to waste the precious space in the row, but maybe there's a good technical reason for this inconvenience? Cheers

EDIT: also it could be enough just to be able to collapse unwanted column (change width to some low value)
